js/main.js

```diff
@@ -101,3 +104,69 @@ var addPinToDom = function () {
 
 addPinToDom();
 
+// Вернуть тип отеля для карточки отеля с описанием
+var ROOM_TYPES = {
+  flat: 'Квартира',
+  bungalo: 'Бунгало',
+  house: 'Дом',
+  palace: 'Дворец'
+};
+
+// Создать шаблон card
+var createMapCard = function (obj) {
+  var mapFilterContainerNode = document.querySelector('.map__filters-container');
+  var cardNode = document.querySelector('#card').content;
+  var popup = cardNode.querySelector('.popup').cloneNode(true);
+
+  popup.querySelector('.popup__title').textContent = obj.offer.title;
+  popup.querySelector('.popup__text--address').textContent = obj.offer.address;
+  popup.querySelector('.popup__text--price').textContent = obj.offer.price + '₽/ночь';
+  popup.querySelector('.popup__type').textContent = ROOM_TYPES[obj.offer.type];
+  popup.querySelector('.popup__text--capacity').textContent = obj.offer.rooms + ' комнаты для ' + obj.offer.guests + ' гостей';
+  popup.querySelector('.popup__text--time').textContent = 'Заезд после ' + obj.offer.checkin + ', выезд до ' + obj.offer.checkout;
+  popup.querySelector('.popup__description').textContent = obj.offer.description;
+  popup.querySelector('.popup__avatar').src = obj.author.avatar;
+  var popupPhotos = popup.querySelector('.popup__photos');
+  var popupPhotosImg = popupPhotos.querySelector('img');
+
+  // Показать только существующие фичи
+  var popupFeatureNodeList = popup.querySelectorAll('.popup__feature');
+  popupFeatureNodeList.forEach(function (item) {
+    item.style.display = 'none';
+  });
+
+  for (var j = 0; j < obj.offer.features.length; j++) {
+
+    if (obj.offer.features[j] === 'wifi') {
+      popup.querySelector('.popup__feature--wifi').style.display = 'inline-block';
+    } else if (obj.offer.features[j] === 'dishwasher') {
+      popup.querySelector('.popup__feature--dishwasher').style.display = 'inline-block';
+    } else if (obj.offer.features[j] === 'parking') {
+      popup.querySelector('.popup__feature--parking').style.display = 'inline-block';
+    } else if (obj.offer.features[j] === 'washer') {
+      popup.querySelector('.popup__feature--washer').style.display = 'inline-block';
+    } else if (obj.offer.features[j] === 'elevator') {
+      popup.querySelector('.popup__feature--elevator').style.display = 'inline-block';
+    } else if (obj.offer.features[j] === 'conditioner') {
+      popup.querySelector('.popup__feature--conditioner').style.display = 'inline-block';
+    }
+  }
```

Comment:
Такое решение тяжело масштабировать, если добавятся или удалятся элементы фич.
Можно сделать универсально и избавиться от `forEach`
```suggestion
  for (var i = 0; i < popupFeatureNodeList.length - obj.offer.features.length; i++) {
    popupFeatureNodeList[popupFeatureNodeList.length - 1 - i].style.display = 'none';
  }
```